### PR TITLE
Added dnstable_query_set_case_sensitive

### PR DESCRIPTION
**Pull request by dvladi77**

### CODE REVIEW

File: dnstable/dnstable.h [hunk 1]
```diff
@@ -107,6 +107,9 @@ dnstable_query_destroy(struct dnstable_query **);
 const char *
 dnstable_query_get_error(struct dnstable_query *);
 
+void
```
Comment (shw700) [inline dnstable/dnstable.h:110]: All other setters follow the convention of returning type dnstable_res.

File: man/dnstable_query.3 [hunk 2]
```diff
@@ -84,6 +84,12 @@ dnstable_query_set_aggregated(struct dnstable_query *\fR\fB\fIq\fR\fR\fB,
 .fi
 .sp
 .nf
+\fBvoid
```
Comment (shw700) [inline man/dnstable_query.3:87]: This is missing its corresponding line in the DESCRIPTION section below (under "dnstable_query_set_...() functions")

File: man/dnstable_query.3 [hunk 3]
```diff
@@ -124,6 +130,8 @@ Once the query object has been constructed, \fBdnstable_query_iter\fR() iterates
 .sp
 \fBdnstable_query_set_aggregated\fR() sets if the results from queries should be aggregated\&. The default is \fItrue\fR for aggregated results\&.
 .sp
+\dnstable_query_set_case_sensitive\fR() sets if the query parameters should be treated as case-sensitive\&. The default is \fIfalse\fR\&.
```
Comment (shw700) [inline man/dnstable_query.3:133]: This line is botched. It should start with \fB

File: man/dnstable_query.3.txt [hunk 4]
```diff
@@ -110,6 +110,9 @@ from the results.  The default is 0, which means to not offset any rows.
 ^dnstable_query_set_aggregated^() sets if the results from queries should be
 aggregated. The default is _true_ for aggregated results.
 
+^dnstable_query_set_case_sensitive() specifies that query parameters should be
+treated as case-sensitive.
+
```
Comment (cmikk) [inline man/dnstable_query.3.txt:115]: This should note that the case is preserved for hostnames added _after_ the call to `dnstable_query_set_case_sensitive()` and will have no effect if called after `dnstable_query_set_data()`

**Review by shw700 — APPROVED** [review]

**Review by cmikk — APPROVED** [review]

**Pull request merged**